### ISSUE TITLE
Grabs a piece of code

### ISSUE THREAD

**Issue by aerohub** [psmb/typo-reporter#3]
Приветствую!
Захватывает код вместе с текстом, (на демо воспроизводится, если выделяются последние слова). Без кода было бы лучше. Спасибо.
![2017-06-27_00001](https://user-images.githubusercontent.com/16353578/27584370-6bbbe0a0-5b40-11e7-886f-ca22e3c1b596.jpg)

UPD (in English): Script grabs a piece of code with the highlighted text. (marked pink on screenshot)

**Comment by dimaip** [comment]
Thanks, I'll reply in English so others could follow. I'm aware of this issue, but didn't get around to fixing it. I'm using `document.getSelection().toString()` to get the highlighted text snippet. Do you have ideas how to get it without contents of script tags?

**Comment by aerohub** [comment]
Well, I have no solution, but may be [this](https://github.com/tarampampam/jquery.textmistake/blob/master/src/jquery.textmistake.js#L197) will help?

**Comment by dimaip** [comment]
Not really, looks like the same thing, just with fallback for compatibility.
